src/components/Forms/MemberForm/MemberForm.jsx

```diff
@@ -296,11 +298,38 @@ export class MemberForm extends Component {
 MemberForm.propTypes = {
   handleClickShowMemberForm: PropTypes.func.isRequired,
   handleClickClearMemberData: PropTypes.func.isRequired,
-  memberData: PropTypes.objectOf(PropTypes.any),
+  memberData: PropTypes.shape({
+    name: PropTypes.string,
+    lastName: PropTypes.string,
+    direction: PropTypes.string,
+    email: PropTypes.string,
+    sex: PropTypes.string,
+    education: PropTypes.string,
+    age: PropTypes.string,
+    universityAverageScore: PropTypes.string,
+    mathScore: PropTypes.string,
+    address: PropTypes.string,
+    mobilePhone: PropTypes.string,
+    skype: PropTypes.string,
+    startDate: PropTypes.string,
+    validated: PropTypes.bool,
+    role: PropTypes.string,
+    userId: PropTypes.string,
+  }),
   isReadOnly: PropTypes.bool.isRequired,
   isEditMode: PropTypes.bool.isRequired,
 };
 
 MemberForm.defaultProps = {
   memberData: null,
 };
+
+function mapStateToProps(state) {
```

Comment:
spread state

Reply:
Thanks, I changed this, and in other files too.
```suggestion
function mapStateToProps({ members: { memberData, isEditMode, isReadOnly } }) {
```